### PR TITLE
cypress test case for scatter chart

### PR DESCRIPTION
**Pull request by nidhisinghai**
Signed-off-by: nidhisinghai <nidhisinghai@persistent.com>

### Description
Cypress test case for scatter chart

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin)

@abasatwar @spattnaik .

### CODE REVIEW

**Review by spattnaik — APPROVED** [review]

File: dashboards-observability/.cypress/integration/VisualizationCharts/10_scatter_chart.spec.js [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+
+/// <reference types="cypress" />
+import {
+    delay,
+    TEST_QUERIES,
+    querySearch,
+    landOnEventVisualizations
+  } from '../../utils/event_constants';
+  
+  const numberOfWindow = 4;
```
Comment (mengweieric) [inline dashboards-observability/.cypress/integration/VisualizationCharts/10_scatter_chart.spec.js:14]: Can we move all these constants to the corresponding constant file under .cypress/utils?

File: dashboards-observability/.cypress/integration/VisualizationCharts/10_scatter_chart.spec.js [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+    beforeEach(() => {
+      renderScatterChart();
+    });
+  
+    it('Render scatter chart and verify by default the data gets render', () => {
+      cy.get('.xy').should('exist');
+    });
+  
+    it('Render scatter chart and verify you see data configuration panel and chart panel', () => {
+      cy.get('.euiPanel.euiPanel--paddingSmall').should('have.length', numberOfWindow);
```
Comment (mengweieric) [inline dashboards-observability/.cypress/integration/VisualizationCharts/10_scatter_chart.spec.js:41]: Same here: can we add data-test-subj?

File: dashboards-observability/.cypress/integration/VisualizationCharts/10_scatter_chart.spec.js (hunk 1 above)
Comment (nidhisinghai) [reply dashboards-observability/.cypress/integration/VisualizationCharts/10_scatter_chart.spec.js:14]: currently for slider and input field we have different default value for different chart (Issue 896), once that gets updated we can move all constants to a common file under .cypress/utils

File: dashboards-observability/.cypress/integration/VisualizationCharts/10_scatter_chart.spec.js (hunk 2 above)
Comment (nidhisinghai) [reply dashboards-observability/.cypress/integration/VisualizationCharts/10_scatter_chart.spec.js:41]: This element don't have data-test-subj defined, only class can be used here

**Review by joshuali925 — APPROVED** [review]

**Review by mengweieric — APPROVED** [review]

**Pull request merged**